test/.eslintrc

```diff
@@ -0,0 +1,2 @@
+rules:
+  import/no-extraneous-dependencies: [error, { devDependencies: true }]
```

Comment:
Fixed in the [eslint-config module](https://github.com/commercetools/eslint-config/pull/9)